### ISSUE TITLE
safe-spawn.js throw object

### ISSUE THREAD

**Issue by thegreyside** [hacksalot/HackMyResume#136]
Hi, I was using 1.5.2 before, and it was OK. Last night I updated to 1.7.4 and then I got this log right after running the `hackmyresume build` command. The problem persists even using the initial `hackmyresume NEW` which provides the default json.

```
C:\Users\Whoami\AppData\Roaming\npm\node_modules\hackmyresume\dist\utils\safe-spawn.js:19
          throw {
          ^
[object Object]
```

The result files were created in out/ folder. But the MSWord can't open the resume.doc, with error message says: "The document cannot be opened because there are problems with the contents."

I don't know what it is since I cannot open it.

I'm using Win7, Nodejs 5.5.0, FRESCA 0.6.0

**Comment by mtantawy** [comment]
+1 same error on Ubuntu 14.04 using hackmyresume 1.7.4

**Comment by yhakong** [comment]
I'm in the same boat too (using mac os x).

**Comment by hacksalot** [comment]
Thanks guys, should be fixed today. This may already be fixed on dev (`npm install hacksalot/hackmyresume#dev`) if you're looking for a quick workaround; will verify in a bit.

**Comment by yhakong** [comment]
Thanks for fixing. Can confirm it works now

**Comment by hacksalot** [comment]
You're welcome! Officially fixed in [1.8.0](https://github.com/hacksalot/HackMyResume/releases/tag/v1.8.0).
